tests/cnf/core/network/policy/tests/multinetpolicy_ipvlan.go

```diff
@@ -0,0 +1,589 @@
+		testPod1, testPod2, testPod3, testPod4, testPod5 *pod.Builder
+		ports                                            = []string{"5001", "5002", "5003"}
+		protocols                                        = []string{"tcp", "tcp", "udp"}
+		allOpen                                          = map[string]string{"5001": "pass", "5002": "pass", "5003": "pass"}
+		allClose                                         = map[string]string{"5001": "fail", "5002": "fail", "5003": "fail"}
+		p5001Open                                        = map[string]string{"5001": "pass", "5002": "fail", "5003": "fail"}
+		p5001p5002Open                                   = map[string]string{"5001": "pass", "5002": "pass", "5003": "fail"}
+	)
+
+	const (
+		testNs1, testNs2             = "policy-ns1", "policy-ns2"
+		pod1, pod2, pod3, pod4, pod5 = "pod1", "pod2", "pod3", "pod4", "pod5"
+	)
+
+	testData := podsData{
+		"pod1": {IPv4: "192.168.10.10/24", IPv6: "2001:0:0:1::10/64", Protocols: protocols, Ports: ports},
+		"pod2": {IPv4: "192.168.10.11/24", IPv6: "2001:0:0:1::11/64", Protocols: protocols, Ports: ports},
+		"pod3": {IPv4: "192.168.10.12/24", IPv6: "2001:0:0:1::12/64", Protocols: protocols, Ports: ports},
+		"pod4": {IPv4: "192.168.20.11/24", IPv6: "2001:0:0:2::11/64", Protocols: protocols, Ports: ports},
+		"pod5": {IPv4: "192.168.20.12/24", IPv6: "2001:0:0:2::12/64", Protocols: protocols, Ports: ports},
+	}
+
+	BeforeAll(func() {
+		By("Verifying if Multi-NetPolicy tests can be executed on given cluster")
+		err := netenv.DoesClusterHasEnoughNodes(APIClient, NetConfig, 1, 1)
+		Expect(err).ToNot(HaveOccurred(),
+			"Cluster doesn't support Multi-NetPolicy test cases as it doesn't have enough nodes")
+
+		By("Listing Worker nodes")
+		workerNodeList, err := nodes.List(
+			APIClient, metav1.ListOptions{LabelSelector: labels.Set(NetConfig.WorkerLabelMap).String()})
+		Expect(err).ToNot(HaveOccurred(), "Failed to list worker nodes")
+
+		By("Fetching SR-IOV interfaces from ENV VAR to use them as master interfaces for ipvlan")
+		sriovInterfacesUnderTest, err = NetConfig.GetSriovInterfaces(2)
+		Expect(err).ToNot(HaveOccurred(), "Failed to retrieve SR-IOV interfaces for testing")
+
+		By("Enable MultiNetworkPolicy support")
+		enableMultiNetworkPolicy(true)
+
+		By("Deploy Test Resources: 2 Namespaces")
+		tNs1, err = namespace.NewBuilder(APIClient, testNs1).WithMultipleLabels(params.PrivilegedNSLabels).
+			WithLabel("ns", "ns1").Create()
+		Expect(err).ToNot(HaveOccurred(), "Failed to create test namespace")
+		tNs2, err = namespace.NewBuilder(APIClient, testNs2).WithMultipleLabels(params.PrivilegedNSLabels).
+			WithLabel("ns", "ns2").Create()
+		Expect(err).ToNot(HaveOccurred(), "Failed to create test namespace")
+
+		By("Deploy Test Resources: 2 NADs for IPVLAN CNI")
+		defineAndCreateIpvlanNAD("ipvlan", testNs1, sriovInterfacesUnderTest[0])
+		defineAndCreateIpvlanNAD("ipvlan", testNs2, sriovInterfacesUnderTest[1])
+
+		By("Deploy Test Resources: 5 Pods")
```

Comment:
It is generally best to write out numbers from zero to 10 in technical writing. So that goes for where ever you used numbers rather then writing them out. Up to you.

Reply:
Ok, Updated accordingly.